docs/application/native/guides/ui/efl/component-gengrid.md

```diff
@@ -24,7 +20,9 @@ To use a gengrid component in your application:
 
 2. Define the gengrid item class:
 
-   1. Create a gengrid item class with the `elm_gengrid_item_class_new()` function, set a style to the item class, and register the callback functions.The gengrid item class must be freed manually with `elm_gengrid_item_class_free()` function after all items are appended.
+   a. Create a gengrid item class with the `elm_gengrid_item_class_new()` function, set a style to the item class, and register the callback functions.  
```

Comment:
Please don't change the number of list item but use 1, 2, ....
Some markdown render doesn't recognize a, b,c , ... as list.